.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
   - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
-  - yarn install --no-lockfile --non-interactive
+  - yarn install --frozen-lockfile --non-interactive
```

Comment:
I think we want to keep this to ensure polly still works with all the latest deps. Otherwise, we won't pick up on breaks until they're reported.

Reply:
Agreed. It looks like node [semver v7](https://github.com/npm/node-semver/blob/bb36c98d71d5760d730abba71c68bc324035dd36/package.json#L36) is no longer supporting node 8 which is breaking our [node 8 build](https://travis-ci.org/Netflix/pollyjs/jobs/635842564#L533). 

Node 8 is EOL as of this year, should we drop support as well? I suspect more and more packages will be doing the same.

Reply:
Yup, agree to drop

Reply:
Sounds good, I'll address in a follow up PR as this is unrelated.